Bump the SDK back up again

dougbu opened the pull request:
Publish in a separate step
- Publishing.props executes the `_GetPackageVersionInfo` target in Microsoft.AspNetCore.App.Runtime.csproj
- that target currently fails when using a new SDK and desktop `msbuild`

Revert "Revert SDK update (#32785)"
- take us back to the 6.0.100-preview.5.21264.3 SDK

This reverts commit 57b9c132a28f1bb2af2b3d4a7d97bc5c6291c6b9.

Reword a comment

dougbu committed:
Publish in a separate step
- Publishing.props executes the `_GetPackageVersionInfo` target in Microsoft.AspNetCore.App.Runtime.csproj
- that target currently fails when using a new SDK and desktop `msbuild`

dougbu committed:
Revert "Revert SDK update (#32785)"
- take us back to the 6.0.100-preview.5.21264.3 SDK

This reverts commit 57b9c132a28f1bb2af2b3d4a7d97bc5c6291c6b9.

dougbu committed:
Reword a comment

dougbu commented:
PR does a few things but first commit won't matter that much after next CI agent rollout. That said, isolation from desktop `msbuild` is _always_ a **Good Thing™️**.

dougbu commented on src/Components/test/E2ETest/Tests/PerformanceTest.cs:
```diff
@@ -33,14 +33,12 @@ protected override void InitializeAsyncCore()
         }
 
         [Fact]
-        [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/32788")]
```
I'm least sure about the changes in this file. @wtgodbe were these quarantined simply due to problems w/ the test app combined w/ the older SDK❔

Mostly wondering: Should we leave these quarantined for 30 days after this gets in i.e. don't un-quarantine as part of this PR❔

dougbu replied:
Thoughts on this❔ I feel like I'm violating @BrennanConroy's new process😉

BrennanConroy replied:
**Our** new process. There are always exceptions to the rule, this wasn't flaky before the SDK issues right?

dougbu replied:
I'm struggling to remember.

wtgodbe replied:
These should be unquarantined, the flakiness was due to the SDK

dougbu commented on eng/Build.props:
```diff
@@ -52,12 +52,9 @@
-                      $(RepoRoot)src\Components\benchmarkapps\Wasm.Performance\TestApp\Wasm.Performance.TestApp.csproj" />
-    <ProjectToExclude Include="
-                      $(RepoRoot)src/Components/test/E2ETestMigration/Microsoft.AspNetCore.Components.Migration.E2ETests.csproj" />    
+                      $(RepoRoot)src/Components/test/E2ETestMigration/Microsoft.AspNetCore.Components.Migration.E2ETests.csproj" />
```
Just noticed this project is mentioned in a couple of *.slnf files but **not** in AspNetCore.sln. Should I fix that solution filter bug in this PR❔

I _think_ we disabled this project because the additional reference to src/Components/test/E2ETest/Microsoft.AspNetCore.Components.E2ETests.csproj caused problems. Is that correct❔

HaoK replied:
Yeah you can leave this disabled for now I will take care of adding this back once my PR is stable, currently the project fails around 33% of the time during build, so its not hard to see if it improves significantly after the SDK is updated

HaoK replied:
In regards to the filter, if it doesn't hurt to add it to the slnf now, that'd be great

dougbu replied:
To be clear: It's AspNetCore.sln that doesn't currently mention Microsoft.AspNetCore.Components.Migration.E2ETests.csproj and the Components.slnf and ComponentsNoDeps.slnf files _should_ be borked. Is anyone seeing issues w/ those solution filters❔

HaoK replied:
Okay yeah I think I saw issues where the project didn't show up before by default, its because it was missing in the AspNetCore sln I guess

dougbu commented:
@dotnet/aspnet-blazor-eng my lower-level questions were mostly for you

dougbu committed:
Add Components.Migration.E2ETests to solution
- used in Components.slnf and ComponentsNoDeps.slnf but hidden because it wasn't here

dougbu commented:
Needs #33019 (thx @BrennanConroy)

BrennanConroy commented:
/azp run

BrennanConroy commented:
Merge when ready. Idk if you want to preserve commits/messages etc.

captainsafia commented:
YAY!
